src/odemis/util/linalg.py

```diff
@@ -262,3 +262,7 @@ def generate_triangulation_points(max_dist: float, area_coords: Iterable[float])
         points = matrix.tolist()
 
     return points
+
+
+def check_overlap(range1: list, range2: list) -> bool:
```

Comment:
"check" is not super clear. I think it'd be nicer to read the calling code if the function is named like `ranges_overlap()`